api/src/test/java/org/pmiops/workbench/featuredworkspace/FeaturedWorkspaceTest.java

```diff
@@ -93,48 +93,34 @@ public void testFeaturedCategory_getFeaturedCategoryForFeaturedWorkspace() {
         .hasValue(FeaturedWorkspaceCategory.TUTORIAL_WORKSPACES);
   }
 
-  void mockFeaturedWorkspaces(String namespace, DbFeaturedCategory dbFeaturedCategory) {
+  void mockFeaturedWorkspaces(
+      String namespace, DbFeaturedCategory dbFeaturedCategory, String firecloudUuid) {
     DbWorkspace mockDbWorkspace =
-        new DbWorkspace().setWorkspaceNamespace(namespace).setWorkspaceId(1);
+        new DbWorkspace()
+            .setWorkspaceNamespace(namespace)
+            .setWorkspaceId(1)
+            .setFirecloudUuid(firecloudUuid);
 
     DbFeaturedWorkspace dbFeaturedWorkspace =
         new DbFeaturedWorkspace().setWorkspace(mockDbWorkspace).setCategory(dbFeaturedCategory);
 
-    RawlsWorkspaceResponse rawlsWorkspaceResponse = new RawlsWorkspaceResponse();
-    rawlsWorkspaceResponse.workspace(new RawlsWorkspaceDetails().workspaceId("1"));
-    rawlsWorkspaceResponse.accessLevel(RawlsWorkspaceAccessLevel.OWNER);
-    when(mockFireCloudService.getWorkspace(
-            mockDbWorkspace.getWorkspaceNamespace(), mockDbWorkspace.getFirecloudName()))
-        .thenReturn(rawlsWorkspaceResponse);
-
-    when(mockFeaturedWorkspaceMapper.toDbFeaturedCategory(
-            FeaturedWorkspaceCategory.valueOf(dbFeaturedCategory.toString())))
-        .thenReturn(dbFeaturedCategory);
     when(mockFeaturedWorkspaceDao.findDbFeaturedWorkspacesByCategory(dbFeaturedCategory))
         .thenReturn(Collections.singletonList(dbFeaturedWorkspace));
-    Workspace mockWorkspace =
-        new Workspace()
-            .namespace(namespace)
-            .featuredCategory(FeaturedWorkspaceCategory.valueOf(dbFeaturedCategory.toString()));
-    when(mockWorkspaceMapper.toApiWorkspace(
-            mockDbWorkspace,
-            rawlsWorkspaceResponse.getWorkspace(),
-            mockInitialCreditsExpirationService))
-        .thenReturn(mockWorkspace);
-    when(mockWorkspaceMapper.toApiWorkspaceResponse(
-            mockWorkspace, rawlsWorkspaceResponse.getAccessLevel()))
-        .thenReturn(
-            new WorkspaceResponse()
-                .workspace(mockWorkspace)
-                .accessLevel(WorkspaceAccessLevel.OWNER));
+
+    rawlsWorkspaces.add(
```

Comment:
each call to this method adds the new one to the list.  At the bottom, the mock then returns the whole list.  yes, it does mean that we're re-mocking that return call each time.